Xwt.XamMac/Xwt.Mac/PanedBackend.cs

```diff
@@ -120,23 +123,54 @@ public override void ReplaceChild (NSView oldChild, NSView newChild)
 
 		public double Position {
 			get {
-				return 0;
+				return position;
 			}
 			set {
+				((CustomSplitView)Widget)?.SetDividerPosition ((nfloat) value);
+				position = value;
+				this.DidResizeSubviews();
 			}
 		}
 		#endregion
 	}
 	
 	class CustomSplitView: NSSplitView, IViewObject
 	{
+		nfloat delayedSetDividerPosition = -1;
+		ViewBackend backend;
```

Comment:
Unused?

Reply:
Yes, backend was only used in an earlier version. Removed. Thanks.